app/grants/tests/factories/cart_activity_factory.py

```diff
@@ -1,18 +1,18 @@
+import random
+
 import factory
 import pytest
-from dashboard.tests.factories.profile_factory import ProfileFactory
+from dashboard.tests.factories import ProfileFactory
 from grants.models.cart_activity import CartActivity
 
 from .grant_factory import GrantFactory
 
 
 @pytest.mark.django_db
 class CartActivityFactory(factory.django.DjangoModelFactory):
-    """Create mock CartActivity for testing."""
```

Comment:
How come were removing all of these descriptions?

Reply:
I removed them because they were technically not correct, these are factories not mocks. I also didn't think they added context. I am happy to put them back in though with the correction that they are factories that return instances of the Django Model.

Reply:
Thats cool with me, I think they're well enough named that we don't need the descriptions!